src/main/java/eggventory/storage/StorageStub.java

```diff
@@ -0,0 +1,108 @@
+package eggventory.storage;
+
+import eggventory.commons.enums.CommandType;
+
+import eggventory.logic.commands.add.AddLoanCommand;
+import eggventory.logic.commands.add.AddPersonCommand;
+import eggventory.logic.commands.add.AddStockCommand;
+import eggventory.logic.commands.add.AddStockTypeCommand;
+import eggventory.logic.commands.add.AddTemplateCommand;
+import eggventory.logic.parsers.ParseAdd;
+import eggventory.model.LoanList;
+import eggventory.model.PersonList;
+import eggventory.model.StockList;
+import eggventory.model.TemplateList;
+
+//@@author patwaririshab
+public class StorageStub {
+
+    /**
+     * Loads stockList from state strings obtained from state stacks.
+     * @param stockListSave string save state of stocks.
+     * @param stockTypeListSave string save state of stocktypes.
+     * @return StockList object.
+     */
+    public StockList loadStockList(String stockListSave, String stockTypeListSave) {
+        StockList savedList = new StockList();
+        try {
+
+            String[] lines2 = stockTypeListSave.split(System.getProperty("line.separator"));
+            for (String line : lines2) {
+                AddStockTypeCommand loadStockTypes = new AddStockTypeCommand(CommandType.ADD, line);
+                loadStockTypes.execute(savedList);
```

Comment:
For loops generally can be slapped into private methods

Reply:
done